github/ipfs.yml

```diff
@@ -18,6 +18,9 @@ members:
     # 1. co-founder of [IPDX](https://ipdx.co), and IPDX is contracted to look after GitHub for this organization.
     # 2. Multiple years of experience managing GitHub organizations of open source projects, including this org.
     - galargh
+    # Why @lidel
+    # Need admin to finish https://github.com/ipfs-shipyard/helia-service-worker-gateway/issues/80
+    - lidel
```

Comment:
I think you need to remove yourself from members for this to work

Reply:
That's correct! It'd be nice if it happened automatically. I made this change now, will be ready for merge soon.